lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -81,6 +81,10 @@ sub create_host_bridge_nm {
         # Re-establish the SSH connection, poo#187197
         reset_consoles;
         select_console('root-ssh');
+        # Set metric the lowest to make br0 always be the default route
+        script_run("nmcli con modify br0 ipv4.route-metric 100");
```

Comment:
Based on your verification result, can this stably resolve the br0 connectivity problem? Just to understand the situation.